py-oxbow/pyproject.toml

```diff
@@ -6,11 +6,9 @@ build-backend = "maturin"
 name = "oxbow"
 dynamic = ["version"]
 description = "Read specialized bioinformatic file formats into dataframes."
-authors = [
-    { name = "Oxbow Developers", email = "nabdennur@gmail.com" }
-]
+authors = [{ name = "Oxbow Developers", email = "nabdennur@gmail.com" }]
 license = { text = "MIT OR Apache-2.0" }
-requires-python = ">=3.10"
+requires-python = ">=3.8"
```

Comment:
Fixes the lower bound of this requirement, but you'll see there are a bunch of common deps that have a higher min python version and require special ignores (see belows)... 

I'd recommend making this at least `>=3.9` since that matches those dependencies and 3.8 is EOL.

Reply:
Agreed.